### DISCUSSION TITLE
Is this project dead?

### DISCUSSION THREAD

**Discussion by topher200** [runsidekick/sidekick#83] [General]
- runsidekick.com does not load
- the discord can't be reached (the invite link was hosted on https://www.runsidekick.com/discord-invitation, which is no longer available)
- the last commit was in January
- there's no python 3.10 version of the python SDK

It's cool if the project is dead, but we should at least add a banner to the README to set expectations. That the links no longer work, etc.

**Comment by suleymanbarman** [comment]
Hi, sorry for the inconvenience about readme.md file. It is updated. Thanks a lot.
